src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAO.scala

```diff
@@ -348,7 +348,7 @@ class PostgresDirectoryDAO(protected val dbRef: DbReference,
                       ${res.name} = ${policyId.resource.resourceId} and
                       ${pol.name} = ${policyId.accessPolicyName}"""
 
-      query.map(rs => rs.get[WorkbenchEmail](g2.email)).single().apply()
+      query.map(rs => rs.get[WorkbenchEmail](groupColumn.email)).single().apply()
```

Comment:
use `g.resultName.email` instead of `groupColumn.email` and on line 339 do `select ${g.result.email}` then remove `groupColumn`